src/models/playlist.js

```diff
@@ -267,7 +267,8 @@ const updatePlaylistField = async (request, reply) => {
         const { ok, n } = response;
 
         if (ok) {
-            return reply({ success: true, message: `${field} was updated` });
+            let updateMessage = (field === 'playlistDate') ? 'The playlist date was updated' : `The document field '${field}' was updated`;
```

Comment:
this rightly won't pass lint, since you should use `const` here. `const` is always preferable to `let` if the variable isn't reassigned and tbh variable reassignment is a bad pattern anyway.